ui/src/main/picsureui/landing/landing.hbs

```diff
@@ -0,0 +1,148 @@
+        transform: scale(1.1);
+        transition: transform .2s;
+    }
+
+    .landing-counts {
+        background: var(--catalyst-blue);
+        color: white;
+        border-radius: 8px;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-around;
+        padding: 3rem 1rem;
+        width: 70%;
+    }
+
+    .landing-count {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        font-size: 2.5rem;
+        font-weight: 100;
+    }
+
+    .search-container {
+        display: flex;
+        align-items: center;
+        margin-bottom: 2rem;
+    }
+
+    .search-container button {
+        margin-left: 1rem;
+        padding: .625rem 2.5rem;
+        border-radius: .625em;
+    }
+
+    #landingSearch {
+        flex-grow: 1;
+        padding: 1rem;
+    }
+
+    .search-container input {
+        padding: 10px;
+        width: 100%;
+        border: 1px solid #C2C2C2;
+    }
+</style>
+<div class="landing-container">
+    <div>
+        <div class="search-container">
+            <input type="text" placeholder="Search studies or variables" class="form-control" style="padding: 10px; width: 100%;"
+                   id="landingSearchInput"/>
+            <button class="btn btn-primary" id="landingSearchButton">Search</button>
+        </div>
+        <div>
+            <p style="font-size: 1.3em"><i>NHLBI AIM-AHEAD® Powered by PIC-SURE</i> can be used to explore studies, conduct feasibility tests, create cohorts,<br /> and select data at the variable-value and genomic variant level.</p>
+        </div>
+    </div>
+    <div class="landing-nav-buttons">
+        <a class="landing-anchor" href="/picsureui/queryBuilder" tabindex="0">
+            <div>
+                <i class="fa-solid fa-magnifying-glass-chart"></i>
+                <p>Search Variables,<br/> Build Cohorts, and<br /> Export Participant-<br/>Level Data in<br/> <b>Authorized Access</b></p>
+            </div>
+        </a>
+        <a class="landing-anchor" href="/picsureui/dataAccess" tabindex="0">
+            <div>
+                <i class="fa-solid fa-table-list"></i>
+                <p>Browse Your<br/>Access to<br/>Studies in<br/><b>Data Access<br/> Dashboard</b></p>
+            </div>
+        </a>
+        <a class="landing-anchor" href="https://biodatacatalyst.nhlbi.nih.gov/" target="_blank" tabindex="0">
+            <div>
+                <i class="fa-solid fa-graduation-cap"></i>
+                <p>Learn<br/> About the <br/><b>BDC Ecosystem</b></p>
+            </div>
+        </a>
+        <a href="https://openpicsure.biodatacatalyst.nhlbi.nih.gov/picsureui/" target="_blank" class="landing-anchor"
+           tabindex="0">
+            <div style="justify-content: center">
+                <i class="fa-solid fa-lock-open"></i>
+                <p>Discover and<br/> Explore All Studies in<br/><b>Open PIC-SURE</b></p>
+            </div>
+        </a>
+    </div>
```

Comment:
Discuss with Danielle: 
- Change Authorized Access to Explorer
- Remove Open PIC-SURE link
- Remove BDC ECO System or link to something more AIM-AHEAD specific